client/containers/DashboardOverview/CollectionOverview/DashboardCollectionOverview.tsx

```
@@ -0,0 +1,178 @@
+		reorderCollectionPubs,
+		addCollectionPub,
+		setCollectionPubContextHint,
+		setCollectionPubIsPrimary,
+		removeCollectionPub,
+	} = useCollectionPubs({ collection, initialCollectionPubs, pubs });
+
+	const renderableCollectionPubs = useMemo(
+		() => collectionPubs.filter((cp) => !!pubsById[cp.pubId]),
+		[collectionPubs, pubsById],
+	);
+
+	useInfiniteScroll({
+		enabled: !isLoading && !includesAllPubs,
+		useDocumentElement: true,
+		onRequestMoreItems: loadMorePubs,
+		scrollTolerance: 100,
+	});
+
+	const handleDragEnd = (dropResult: DropResult) => {
+		const { source, destination } = dropResult;
+		if (destination) {
+			reorderCollectionPubs(source.index, destination.index);
+		}
+	};
+
+	const handleAddCollectionPub = (pub: PubWithCollections) => {
+		setPubsAddedToCollection((current) => [...current, pub]);
+		addCollectionPub(pub);
+	};
+
+	const renderCollectionPubRow = (
+		collectionPub: CollectionPub,
+		dragHandleProps: Maybe<DraggableProvidedDragHandleProps>,
+		isDragging: boolean,
+	) => {
+		const pub = pubsById[collectionPub.pubId]!;
+		return (
+			<PubOverviewRow
+				className={classNames(isDragging && 'collection-overview-row-is-dragging')}
+				pub={pub}
+				leftIconElement={!isSearching && <DragHandle dragHandleProps={dragHandleProps} />}
+				rightElement={
+					<PubMenu
+						pub={pub}
+						collection={collection}
+						collectionPub={collectionPub}
+						setCollectionPubContextHint={setCollectionPubContextHint}
+						setCollectionPubIsPrimary={setCollectionPubIsPrimary}
+						removeCollectionPub={removeCollectionPub}
+					/>
+				}
+			/>
+		);
+	};
+
+	const renderEmptyState = () => {
+		if (pubs.length === 0 && hasLoadedAllPubs) {
+			if (isSearching) {
+				return <SpecialRow>No matching Pubs.</SpecialRow>;
+			}
+			return (
+				<NonIdealState
+					className="collection-empty-state"
+					icon="clean"
+					title="This Collection looks brand new!"
+					description="Try creating a Pub or Collection from above."
```

This default language seems a bit misleading. Perhaps just, "Try creating a Pub using the button above"

---

Oh! Just copied that from the Community side. Nice catch.